packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -114,24 +160,25 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
             },
             (isSuggesting: boolean) => {
                 this.setIsSuggesting(isSuggesting);
-            },
-            editor
+            }
         );
     }
 
     /**
      * Dispose this plugin
      */
     public dispose() {
-        this.editor = null;
-        this.isSuggesting = null;
-        this.blockSuggestions = null;
-        this.eventHandledOnKeyDown = null;
+        this.isSuggesting = false;
+        this.blockSuggestions = false;
+        this.eventHandledOnKeyDown = false;
         this.lastKnownRange = null;
-        this.isPendingInputEventHandling = null;
-        this.currentInputLength = null;
-        this.newInputLength = null;
+        this.isPendingInputEventHandling = false;
+        this.currentInputLength = undefined;
+        this.newInputLength = undefined;
         this.dataProvider.onDispose();
+        this.editor = null!;
+        this.dataProvider = null!;
+        this.pickerOptions = null!;
```

Comment:
Please add comment to explain why we need to and it is safe to set to "null!" here